Documentation/Visualizations/Map.md

```diff
@@ -0,0 +1,95 @@
+
+### Using Country/Region option
+1. Switch the workbook to edit mode by clicking on the _Edit_ toolbar item.
+2. Use the _Add query_ link to add a log query control to the workbook.  
+3. Select the query type as _Log_, resource type as Application Insights and then pick any Application Insights resource which has pageViews data.
+4. Use the Query editor to enter the KQL for your analysis and click on Run Query afterwards
+    ```
+    pageViews
+    | project duration, itemCount, client_CountryOrRegion
+    | limit 20
+    ```
+5. Set size values to `Large`
+6. Set the visualization to _Map_
+7. All the settings will be auto-populated. For custom settings, click on _Map Settings_ button to open the settings pane.
+
+
+## Map Settings has three sub-section:
+![Map Settings](../Images/mapSettings.png)
+
+## Layout Settings
+
+| Setting | Explanation |
+|:------------- |:-------------|
+| `Location Info Using` | Select a way to get location of items shown on the map. <ul><li> **Latitude/Longitude**: Select this options if there are columns with latitude and longitude information. Each row with latitude and longitude data will be shown as distinct item on the map. </li> <li>**Azure location**: Select this options if there is a column which has Azure Location(eastus, westeurope, centralindia, etc.) information. Specify that column and we will fetch the corresponding latitude and longitude for each Azure location, group same location rows(based on Aggregation specified) together to show the locations on the map. </li><li>**Azure resource**: Select this options if there is a column which has Azure resource information(storage account, cosmosdb account, etc). This is how the Azure resource looks: "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}. Specify that column and we will fetch the corresponding latitude and longitude for each Azure resource, group same location(Azure location) rows(based on Aggregation specified) together to show the locations on the map. </li> <li>**Country/Region**: Select this options if there is a column which has country/region name/code (US, United States, IN,  India, CN, China) information. Specify that column and we will fetch the corresponding latitude and longitude for each Country/Region/Code and group the rows together with same Country-Region Code/Country-Region name to show the locations on the map. Note: Country Name and Country code won't be grouped together as a single entity on the map. </li> </ul>|
+| `Latitude/Longitude` | These two option will be visible if Location Info field value is: Latitude/Longitude. Select the column which has latitude in the latitude field and longitude in the longitude field respectively. |
+| `Azure location field` | This option will be visible if Location Info field value is: Azure location. Select the column which the Azure location information. |
```

Comment:
for azure location, you might want to link out to azure docs for these values are to make it obvious that it is those specific kinds of values?

Reply:
done